troubleshooting/troubleshooting-cannot-receive-push-notifications-on-android-when-the-app-is-closed.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Cannot Receive Push Notifications on Android When the App Is Closed
+description: On Android, push notifications are not received when the app is not running (closed). Notifications are received when the app is running either in the foreground or in the background.  
+type: troubleshooting
+page_title:  Push Notifications are not received on Android When the App Is Closed 
+slug: cannot-receive-push-notifications-on-android-when-the-app-is-closed
+position:
+tags: push, Android, push notifications, GCM, Xiaomi, Huawei, MIUI
+teampulseid: 
+ticketid: 1052512, 1051939
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Notifications (Backend Services)</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Push Notifications</td>	
+  </tr>
+</table>
+
+## Description
+
+On Android devices from some vendors push notifications sent through {{site.tp}} are not received when the app is closed. Push notifications are received when the app is opened in the foreground or running in the background.
+
+## Cause
+
+On these devices most of the apps are not permitted by default to be awaken in the background so that they can process the received push notification. 
+
+At the time of writing the identified devices that are showing this behavior are:
```

Comment:
Currently, this behavior is observed only on devices manufactured by:

Just a suggestion. The one you have is good as well :)